### PR TITLE
Fix broken links

### LINKED ISSUE #1413: Fix broken links
The following links are possibly broken:
- [x] https://faucet.testnet.shimmer.network/api/enqueue
- [x] https://www.etogruppe.com/
- [x] https://selv.iota.org/
- [x] https://identity.foundation/JcsEd25519Signature2020/
- [x] https://todo_link_to_inx_page/
- [x] https://linux.die.net/man/1/crontab
- [x] https://explorer.tanglebay.com/mainnet
- [x] https://api.testnet.shimmer.network/
- [x] https://crates.io/crates/iota-sdk
- [x] https://ed25519.cr.yp.to.md/
- [x] https://www.researchgate.net/publication/220854689_Testing_patterns_for_software_transactional_memory_engines
- [x] https://deps.rs/repo/github/iotaledger/stronghold.rs
- [x] https://dashboard.evm.testnet.shimmer.network/
- [x] https://toolkit.evm.testnet.shimmer.network/
- [x] https://erc20-faucet.evm.testnet.shimmer.network/
- [x] https://explorer.wasp.sc.iota.org/
- [x] https://www.justaskgemalto.com/en/what-is-digital-identity/
- [x] https://www.apkmirror.com/apk/google-inc/android-system-webview/android-system-webview-96-0-4664-104-release/android-system-webview-96-0-4664-104-4-android-apk-download/
- [x] https://bitvavo.com/
- [x] https://faucet.testnet.shimmer.network.md/
- [x] https://twitter.com/iota/
- [x] https://www.reddit.com/r/Iota/
- [x] https://evm-toolkit.evm.shimmer.network/
- [x] https://www.twitch.tv/iotadev/
- [x] https://discord-layerzero.netlify.app/discord
- [x] https://discord.shimmer.network/
- [x] https://www.tangletreasury.org/governance-and-treasury-minutes
- [x] https://iotatreasury.org/
- [x] https://layerzeroscan.com/
- [x] https://bloomwallet.io/
- [x] https://youtube.com/playlist?list=PL5joP0FyJQU4eKPWfQWradNVjMIPq86ID
- [x] https://iotaledger.github.io/firefly/specifications/governance/
- [x] https://ieeexplore.ieee.org/document/502236
- [x] https://arxiv.org/pdf/2009.04987
- [x] https://files.iota.org/papers/IOTA_2.0_Incentives_And_Tokenomics_Whitepaper.pdf
- [x] https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/inx-chronicle/main/documentation/api/api-explorer.yml
- [x] https://jwt.io/
- [x] https://docs.layerzero.network/
- [x] https://www.gnu.org/licenses/gpl-3.0.en.html
- [x] https://shimmergov.notion.site/18af77a6634f4d9d91b7c16d9f294fbd?v=dcd7703eac1e4dd3b5302bf533546cb3
- [x] https://meta.discourse.org/t/how-to-create-polls/77548
- [x] https://seemly-seer-d1b.notion.site/f11f6bb9a4ee4f90855794dfd8d28927?v=a6eb959920f54452a4e2dd13e21e6104
- [x] https://assets.ctfassets.net/r1dr6vzfxhev/2t4uxvsIqk0EUau6g2sw0g/45eae33637ca92f85dd9f4a3a218e1ec/iota1_4_3.pdf
- [x] https://explorer.testnet.shimmer.network/testnet
- [x] https://opensource.org/licenses/MIT
- [x] https://docs.libp2p.io/concepts/peer-id/
- [x] https://ipfs.tech/
- [x] https://grpc.io/
- [x] https://curl.se/
- [x] https://www.json.org/
- [x] https://hub.docker.com/r/iotaledger/inx-api-core-v0
- [x] https://www.coingecko.com/en/coins/iota
- [x] https://www.apache.org/
- [x] https://prometheus.io/
- [x] https://grafana.com/
- [x] https://www.mongodb.com/
- [x] https://www.conventionalcommits.org/en/v1.0.0/
- [x] http://www.haproxy.org/
- [x] https://www.nginx.com/
- [x] https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord
- [x] https://helpwithpenny.com/export-and-import-metamask-private-key/
- [x] https://remix-project.org/
- [x] https://traefik.io/
- [x] https://docs.ethers.org/v5/
- [x] https://reactjs.org/
- [x] https://www.linkedin.com/company/iotafoundation/
- [x] https://nodejs.org/en
- [x] https://docusaurus.io/docs/markdown-features
- [x] https://git-scm.com/book/en/v2/Git-Basics-Tagging
- [x] https://wiki.iota.org/shimmer/iota-sdk/references/nodejs/classes/Client/
- [x] https://docusaurus.new/
- [x] https://github.com/jlvandenhout
- [x] https://explorer.iota.org/shimmer
- [x] https://www.vda.de/en/services/Publications/vda-5530-abv-according-to-iso-19987-%28epcis%29-v1.0-september-2017.html
- [x] https://identity.foundation/JcsEd25519Signature2020/#ProofGeneration
- [x] https://linux.die.net/man/1/rsync
- [x] https://crates.io/crates/iota_stronghold
- [x] https://twitter.com/shimmernet
- [x] https://svelte.dev/tutorial/
- [x] https://www.zebra.com/es/es/solutions/intelligent-edge-solutions/savanna.html
- [x] https://identity.foundation/didcomm-messaging/spec/
- [x] https://crates.io/crates/stronghold-derive
- [x] https://linux.die.net/man/1/scp
- [x] https://github.com/Critical94
- [x] https://twitter.com/i/flow/signup
- [x] https://pypi.org/project/iota-sdk.md
- [x] https://crates.io/crates/stronghold-engine
- [x] https://identity.foundation/
- [x] https://identity.foundation/didcomm-messaging/spec/#protocols
- [x] https://tools.ietf.org/id/draft-msporny-base58-01.html
- [x] https://crates.io/crates/stronghold-runtime
- [x] https://twitter.com/i/lists/1564924491850989575
- [x] https://crates.io/crates/stronghold-utils
- [x] https://identity.foundation/didcomm-messaging/spec/#connections
- [x] https://identity.foundation/didcomm-messaging/spec/#didcomm-signed-message
- [x] https://docs.rs/iota-sdk/latest/iota_sdk//wallet/account/struct.Account.html#method.sync
- [x] https://twitter.com/iota
- [x] https://identity.foundation/didcomm-messaging/spec/#tamper-resistant-oob-messages
- [x] https://identity.foundation/didcomm-messaging/spec/#defined-profiles
- [x] https://twitter.com/iota_dev
- [x] https://identity.foundation/didcomm-messaging/spec/#routing
- [x] https://identity.foundation/didcomm-messaging/spec/#message-headers
- [x] https://developer.zebra.com/user/me/apps
- [x] https://twitter.com/ShimmerGov
- [x] https://identity.foundation/didcomm-messaging/spec/#attachments
- [x] https://tools.ietf.org/id/draft-rundgren-json-canonicalization-scheme-00.html
- [x] https://identity.foundation/didcomm-messaging/spec/#standard-message-encoding
- [x] https://identity.foundation/didcomm-messaging/spec/#sender-authenticated-encryption
- [x] https://identity.foundation/didcomm-messaging/spec/#anonymous-encryption
- [x] https://identity.foundation/.well-known/resources/did-configuration/
- [x] https://wiki.iota.org/tips/tips/TIP-0045/
- [x] https://identity.foundation/didcomm-messaging/spec/#threads
- [x] https://identity.foundation/didcomm-messaging/spec/#acks
- [x] https://wiki.iota.org/tips/tips/TIP-0022/tip-0022.md#detailed-design
- [x] https://identity.foundation/presentation-exchange/spec/v1.0.0/
- [x] https://identity.foundation/didcomm-messaging/spec/#problem-reports
- [x] https://wiki.iota.org/tips/tips/TIP-0032/tip-0032.md#global-parameters
- [x] https://identity.foundation/.well-known/resources/did-configuration/#did-configuration-resource
- [x] https://identity.foundation/.well-known/resources/did-configuration/#linked-domain-service-endpoint
- [x] https://wiki.iota.org/tips/tips/TIP-0046/tip-0046.md
- [x] https://identity.foundation/.well-known/resources/did-configuration/#did-configuration-uri
- [x] https://identity.foundation/.well-known/resources/did-configuration/#did-configuration-resource-verification
- [x] https://identity.foundation/.well-known/resources/did-configuration/#json-web-token-proof-format
- [x] https://wiki.iota.org/tips/tips/TIP-0046/
- [x] https://identity.foundation/.well-known/resources/did-configuration/#domain-linkage-credential
- [x] https://wiki.iota.org/tips/tips/TIP-0041/tip-0041.md#basic-output
- [x] https://wiki.iota.org/tips/tips/TIP-0042/tip-0042.md#account-output
- [x] https://wiki.iota.org/tips/tips/TIP-0044/tip-0044.md#foundry-output
- [x] https://wiki.iota.org/tips/tips/TIP-0043/tip-0043.md#nft-output
- [x] https://wiki.iota.org/tips/tips/TIP-0040/tip-0040.md#delegation-output
- [x] https://wiki.iota.org/tips/tips/TIP-0045/tip-0045#signature-unlock
- [x] https://wiki.iota.org/tips/tips/TIP-0045/tip-0045#reference-unlock
- [x] https://wiki.iota.org/tips/tips/TIP-0042/tip-0042#account-locking--unlocking
- [x] https://wiki.iota.org/tips/tips/TIP-0043/tip-0043#nft-locking--unlocking
- [x] https://wiki.iota.org/build/wasp-wasm/how-accounts-work/
- [x] https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md
- [x] https://docs.rs/stronghold_engine/engine/latest/store/index.html
- [x] https://docs.rs/stronghold_engine/engine/latest/vault/index.html
- [x] https://github.com/decentralized-identity/didcomm-messaging/blob/ef997c9d3cd1cd24eb182ffa2930a095d3b856a9/docs/spec-files/feature_discovery.mdx
- [x] https://github.com/decentralized-identity/didcomm-messaging/blob/9039564e143380a0085a788b6dfd20e63873b9ca/docs/spec-files/trustping.mdx
- [x] https://github.com/decentralized-identity/didcomm-messaging/blob/9039564e143380a0085a788b6dfd20e63873b9ca/docs/spec-files/feature_discovery.mdx
- [x] https://github.com/iotaledger/identity.rs/tree/dev/bindings/stronghold-nodejs
- [ ] https://github.com/iotaledger/iota.go/tree/v3
- [x] https://github.com/iotaledger/tips/blob/token-metadata-schema/tips/TIP-0030/tip-0030.md
- [x] https://github.com/iotaledger/iota-sdk/tree/develop/sdk/sdk/Cargo.toml
- [x] https://github.com/iotaledger/stronghold.rs/blob/dev/documentation/docs/meta/Audit.pdf
- [x] https://github.com/iotaledger/wasp/blob/develop/documentation/docs/misc/coretypes.md
- [x] https://github.com/iota-wiki/iota-wiki/tree/main/shimmer/learn/governance
- [x] https://github.com/iotaledger/inx-chronicle/compare/v1.0.0-beta.17...1.0.0-beta.18
- [x] https://github.com/iotaledger/inx-chronicle/commit/ebd200cb4b7e8db425148b91c9fe832d9c54522a

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Fixed the broken links mentioned by the workflow.

## Links to any relevant issues

Fixes #1413, #1302, #1320, #1333, #1339, #1358, #1374, #1382, #1390, #1393, #1395, #1401, #1405

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Dr-Electron** [comment]
I fixed all links excepts some for which I will explain why I didn't:

Those links won't need fixing as the docs will be either removed or updated in the near future anyway.

# Identity 0.6:
@eike-hass fyi
- https://selv.iota.org/
- https://identity.foundation/JcsEd25519Signature2020/
- https://tools.ietf.org/id/draft-rundgren-json-canonicalization-scheme-00.html
- https://tools.ietf.org/id/draft-msporny-base58-01.html
- https://tools.ietf.org/id/draft-rundgren-json-canonicalization-scheme-00.html
- https://github.com/decentralized-identity/didcomm-messaging/blob/ef997c9d3cd1cd24eb182ffa2930a095d3b856a9/docs/spec-files/feature_discovery.mdx
- https://github.com/decentralized-identity/didcomm-messaging/blob/9039564e143380a0085a788b6dfd20e63873b9ca/docs/spec-files/trustping.mdx
- https://github.com/decentralized-identity/didcomm-messaging/blob/9039564e143380a0085a788b6dfd20e63873b9ca/docs/spec-files/feature_discovery.mdx
- https://github.com/iotaledger/identity.rs/tree/dev/bindings/stronghold-nodejs
 
# Zebra:
@jmcanterafonseca-iota fyi
- https://www.justaskgemalto.com/en/what-is-digital-identity/
- https://www.vda.de/en/services/Publications/vda-5530-abv-according-to-iso-19987-%28epcis%29-v1.0-september-2017.html
- https://svelte.dev/tutorial/
- https://www.zebra.com/es/es/solutions/intelligent-edge-solutions/savanna.html
- https://developer.zebra.com/user/me/apps

There are also some broken link in a TIP related to iota.go. I will contact the DRI to fix it.

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ You can use Gitpod (a free, online, VS Code-like IDE) for contributing. With a s
 Phylo - [Phyloiota](https://github.com/Phyloiota) - Phylo [Community DAO - lets go!]#2233  
 Jeroen van den Hout - [jlvandenhout](https://github.com/jlvandenhout) - jvdhout#4402  
 Dr.Electron - [Dr-Electron](https://github.com/Dr-Electron) - Dr.Electron#9370  
-Critical - [Critical94](https://github.com/Critical94) - Critical#7111  
+Critical - [Critical94](https://github.com/kilianhln) - Critical#7111
```
Comment (lucas-tortora) [inline README.md:119]: Shouldn't this be:
```suggestion
Kilian Helène - [kilianhln](https://github.com/kilianhln) - Critical#7111  
```

**Review by lucas-tortora — APPROVED** [review]
Just a small doubt on my end :)

File: README.md (hunk 1 above)
Comment (Dr-Electron) [reply README.md:119]:
```suggestion
Critical - [kilianhln](https://github.com/kilianhln) - Critical#7111  
```

**Pull request merged**